composer.json

```diff
@@ -38,6 +38,7 @@
         "symfony/validator": "^2.2",
         "symfony/form": "~2.1",
         "symfony/filesystem": "^2.1",
+        "symfony/expression-language": "^2.6|^3.0",
```

Comment:
You add some dependency to `composer.json` but not to `composer.lock`, so I pull the latest master, run `composer install` & then run `./vendor/bin/phpunit`:
> Fatal error: Class 'Symfony\Component\ExpressionLanguage\ExpressionFunction' not found in .../tests/JMS/Serializer/Tests/Serializer/BaseSerializationTest.php on line 243

I think this should be fixed.